rules/windows/lateral_movement_dcom_hta.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of Distributed Component Object Model (DCOM) to execute commands from a remote host, which are
+launched via the HTA Application COM Object. This behavior may indicate an attacker abusing a DCOM application to move
+laterally while attempting to evading detection.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Incoming DCOM Lateral Movement with MSHTA"
+references = ["https://codewhitesec.blogspot.com/2018/07/lethalhta.html"]
+risk_score = 73
+rule_id = "622ecb68-fa81-4601-90b5-f8cd661e4520"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence with maxspan=1m
+  [process where event.type in ("start", "process_started") and
+   process.name == "mshta.exe" and process.args == "-Embedding"
+  ] by host.id, process.entity_id
+  [network where event.type == "start" and process.name=="mshta.exe" and 
+   event.action == "connection_accepted" and network.direction == "incoming" and network.transport == "tcp" and
+   source.port > 49151 and destination.port > 49151
+  ] by host.id, process.entity_id
+'''
```

Comment:
`event.action` would prevent this from triggering with winlogbeat here, but I think the logic should be fine without it

```suggestion
query = '''
sequence with maxspan=1m
  [process where event.type in ("start", "process_started") and
     process.name : "mshta.exe" and process.args : "-Embedding"
  ] by host.id, process.entity_id
  [network where event.type == "start" and process.name : "mshta.exe" and 
     network.direction == "incoming" and network.transport == "tcp" and
     source.port > 49151 and destination.port > 49151
  ] by host.id, process.entity_id
'''
```

Reply:
its ok as long as the incoming direction and both ports are higher than 49151 preserved: 

![image](https://user-images.githubusercontent.com/64742097/100727265-49d26b00-33c6-11eb-8dea-48a03471b44f.png)